blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -558,13 +519,13 @@ export const actions = {
                 // We're only interested in this event if we're already managing branch state
                 // associated with this multi-branch job.
                 if (!multibranchPipelines[pipelineName]) {
-                    return;
+                    return Promise.resolve(null);
```

Comment:
Pretty sure you need a polyfill for this not to blow up on some versions of IE (as late as IE 11?) My pagination PR added this polyfill, if you want to see one that seems to work